File: app/src/main/java/com/jcs/where/search/adapter/SearchHistoryAdapter.java
package com.jcs.where.search.adapter;

import android.widget.TextView;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.viewholder.BaseViewHolder;
import com.jcs.where.R;

import org.jetbrains.annotations.NotNull;

/**
 * create by zyf on 2021/1/31 2:33 下午
 */
public class SearchHistoryAdapter extends BaseQuickAdapter<String, BaseViewHolder> {

    public SearchHistoryAdapter() {
        super(R.layout.item_searchhistory);
    }

    @Override
    protected void convert(@NotNull BaseViewHolder baseViewHolder, String s) {
        TextView searchHistoryTv = baseViewHolder.findView(R.id.tv_searchhistory);
        searchHistoryTv.setText(s);
    }
}

File: app/src/main/java/com/jcs/where/features/store/history/SearchHistoryAdapter.kt
package com.jcs.where.features.store.history

import android.widget.TextView
import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.viewholder.BaseViewHolder
import com.jcs.where.R

/**
 * Created by Wangsw  2021/6/7 14:31.
 *
 */
class SearchHistoryAdapter:BaseQuickAdapter<String , BaseViewHolder>(R.layout.item_search_history) {

    override fun convert(holder: BaseViewHolder, item: String) {
        val content_tv = holder.getView<TextView>(R.id.content_tv)
        content_tv.text = item
    }
}